Classes/DataProcessing/GridProcessor.php

```diff
@@ -65,12 +65,12 @@ public function process(
 
         $processorConfiguration = array_merge($this->defaultProcessorConfiguration, $processorConfiguration);
 
-        $textPositionVertical = $processedData['gallery']['position']['vertical'];
-        $textPositionHorizontal = $processedData['gallery']['position']['horizontal'];
+//        $textPositionVertical = $processedData['gallery']['position']['vertical'];
```

Comment:
Please remove lines 68+69, we don't seem to need them. Then amend the commit and force-push.